pkg/sql/virtual_schema.go

```diff
@@ -176,6 +177,7 @@ func (t virtualSchemaTable) initVirtualTableDesc(
 		tree.PersistencePermanent,
 	)
 	if err != nil {
+		log.Errorf(ctx, "initVirtualDesc problem: %v\n%s", err, t.schema)
```

Comment:
Do we need this extra here? Shouldn't the return err handle it, we can also wrap the error instead.

Reply:
The problem of not having these is that the error is eventually wrapped and manipulated later in the code. During my tests I end up having a bad create table that I was unable to find which definition until I did some debugging of the code.

I strongly recommend to leave that log, in a production environment I don't believe it gets printed.